phoenix-core/src/test/java/org/apache/phoenix/util/ScanUtilTest.java

```diff
@@ -45,291 +48,305 @@
-                PChar.INSTANCE.toBytes("b"),
-                Bound.UPPER
-                ));
-        // 15, Upper bound, range inclusive, single inclusive, increment at end.
-        testCases.addAll(
-            foreach(new KeyRange[][]{{
-                    PChar.INSTANCE.getKeyRange(Bytes.toBytes("a"), true, Bytes.toBytes("b"), true),},{
-                    PChar.INSTANCE.getKeyRange(Bytes.toBytes("1"), true, Bytes.toBytes("1"), true),}},
-                new int[] {1,1},
-                PChar.INSTANCE.toBytes("b2"),
-                Bound.UPPER
-                ));
-        // 16, Upper bound, single inclusive, range exclusive, no increment at end.
-        testCases.addAll(
-            foreach(new KeyRange[][]{{
-                    PChar.INSTANCE.getKeyRange(Bytes.toBytes("a"), true, Bytes.toBytes("a"), true),},{
-                    PChar.INSTANCE.getKeyRange(Bytes.toBytes("1"), true, Bytes.toBytes("2"), false),}},
-                new int[] {1,1},
-                PChar.INSTANCE.toBytes("a2"),
-                Bound.UPPER
-                ));
-        // 17, Upper bound, unbound key, fixed length;
-        testCases.addAll(
-            foreach(new KeyRange[][]{{
-                    PChar.INSTANCE.getKeyRange(Bytes.toBytes("a"), true, Bytes.toBytes("a"), true),},{
-                        KeyRange.EVERYTHING_RANGE,}},
-                new int[] {1,1},
-                PChar.INSTANCE.toBytes("b"),
-                Bound.UPPER
-                ));
-        // 18, Upper bound, unbound key, variable length;
-        testCases.addAll(
-            foreach(new KeyRange[][]{{
-                    PChar.INSTANCE.getKeyRange(Bytes.toBytes("a"), true, Bytes.toBytes("a"), true),},{
-                    KeyRange.EVERYTHING_RANGE,}},
-                new int[] {1,1},
-                PChar.INSTANCE.toBytes("b"),
-                Bound.UPPER
-                ));
-        // 19, Upper bound, keys wrapped around when incrementing.
-        testCases.addAll(
-            foreach(new KeyRange[][]{{
-                PChar.INSTANCE.getKeyRange(new byte[] {-1}, true, new byte[] {-1}, true)},{
-                PChar.INSTANCE.getKeyRange(new byte[] {-1}, true, new byte[] {-1}, true)}},
-                new int[] {1, 1},
-                ByteUtil.EMPTY_BYTE_ARRAY,
-                Bound.UPPER
-                ));
-        // 20, Variable length
-        testCases.addAll(
-            foreach(new KeyRange[][]{{
-                    PChar.INSTANCE.getKeyRange(Bytes.toBytes("a"), true, Bytes.toBytes("a"), true),},{
-                    PVarchar.INSTANCE.getKeyRange(Bytes.toBytes("A"), true, Bytes.toBytes("B"), true),}},
-                new int[] {1,0},
-                ByteUtil.nextKey(ByteUtil.concat(PVarchar.INSTANCE.toBytes("aB"), QueryConstants.SEPARATOR_BYTE_ARRAY)),
-                Bound.UPPER
-                ));
-        return testCases;
-    }
+        @Parameters(name = "{0} {1} {2} {3} {4}") public static Collection<Object> data() {
```

Comment:
nit: same as above. Could you also add parameter names corresponding to each parameter like this: https://github.com/apache/phoenix/blob/master/phoenix-core/src/it/java/org/apache/phoenix/tx/ParameterizedTransactionIT.java#L104

Reply:
I did the line but I think naming those are outside my scope as I did not touch any of that code.

Reply:
Got it. I would do it just to make the code more readable one commit at a time. Thanks!